app/uk/gov/hmrc/tai/connectors/RtiConnector.scala

```diff
@@ -63,7 +63,7 @@ class RtiConnector @Inject()(
 
   def getPaymentsForYear(nino: Nino, taxYear: TaxYear)(
     implicit hc: HeaderCarrier): Future[Either[RtiPaymentsForYearError, Seq[AnnualAccount]]] =
-    if (rtiToggle.rtiEnabled) {
+    if (rtiToggle.rtiEnabled && taxYear.year < TaxYear().next.year) {
       val NGINX_TIMEOUT = 499
```

Comment:
We may need a log statement here capturing the RTI call with year. This is to capture the API call evidence